projects/json-c/tokener_parse_ex.dict

```diff
@@ -0,0 +1,27 @@
+"{"
+"}"
+","
+"["
+"]"
+","
+":"
+"e"
+"e+"
+"e-"
+"E"
+"E+"
+"E-"
+"\""
+"\\"
+"\/"
```

Comment:
This dictionary does not appear to be valid, are you sure it's being used?

```
ParseDictionaryFile: error in line 16
		"\/"
```

Reply:
I'm not and given that you've found this, I'd expect that it isn't. How did you notice this error?

Reply:
I copied it for #1119. Look for `-dict=` in the command line that is printed at the beginning of the `run_fuzzer` output.

Reply:
:+1: thanks